src/main/java/org/jabref/gui/util/FileFilterConverter.java

```diff
@@ -27,7 +27,11 @@ private FileFilterConverter() {
     }
 
     public static FileChooser.ExtensionFilter toExtensionFilter(FileType fileType) {
-        String description = Localization.lang("%0 file", fileType.toString());
+        String description = Localization.lang(
+                "%0 file (%1)",
+                fileType.getName(),
+                String.join(", ", fileType.getExtensionsWithAsteriskAndDot())
+        );
```

Comment:
It is not explicit in https://devdocs.jabref.org/code-howtos/localization.html, but `Localization.lang(` has to be in the same line. - Our check reads the java files as text (and does not a full JavaParsing, because JavaParser cannot parse full modern Java: https://github.com/javaparser/javaparser/issues/3364). Historical reference: https://github.com/JabRef/jabref/pull/7279

Thus, please rewrite

```suggestion
        String fileList = String.join(", ", fileType.getExtensionsWithAsteriskAndDot());
        String description = Localization.lang("%0 file (%1)", fileType.getName(), fileList);
```